### ISSUE TITLE
Pi face was working and now is not

### ISSUE THREAD

**Issue by mattcook46** [piface/pifacedigitalio#26]
My piface was working. Then I dropped my motor and cables were disconnected from one of the 2 relays on the board.   Now I can't seem to control my board.  I use the emulator but it seems to have no effect on my outputs.  No LEDs light up on demand anymore.  Is there a way to determine if my board is ruined?.... A way to test it besides the emulator / Python.

**Comment by tompreston** [comment]
You may have shorted a connection when the cables came out. I doubt this is a software issue but it might be worth doing an update and see if that helps. Also, what happens when you run this code:

    import pifacedigitalio
    pfd = pifacedigitalio.PIFaceDigital()
    pfd.output_pins[0].turn_on()

Do you get any exceptions?

**Comment by mattcook46** [comment]
Yes it has errors on the second command now...

AttributeError: 'module' object has no attribute 'PIFaceDigital'

Sent from my iPad

On Feb 16, 2015, at 3:48 AM, Thomas Preston <notifications@github.com<mailto:notifications@github.com>> wrote:


You may have shorted a connection when the cables came out. I doubt this is a software issue but it might be worth doing an update and see if that helps. Also, what happens when you run this code:

import pifacedigitalio
pfd = pifacedigitalio.PIFaceDigital()
pfd.output_pins[0].turn_on()


Do you get any exceptions?

**Comment by tompreston** [comment]
Sorry I made a typo:

    PiFaceDigital

with a lower-case ``i`` after first ``P``.

More examples are in the docs: http://pifacedigitalio.readthedocs.org/example.html

**Comment by mattcook46** [comment]
Ya,
That was why it didn't work the first time.  Once I made the correction in spelling, I didn't get any errors.  But the next command:

Import pifacedigitalio
pfd = pifacedigitalio.PiFaceDigital()
pfd.output_pins[0].value = 1

Returns no errors but no light comes on.... No click....

I've tried different output pins.... Still not turning on.  That particular code was not returning any errors.

Any other ideas?

Sent from my iPhone

On Feb 19, 2015, at 4:33 AM, Thomas Preston <notifications@github.com<mailto:notifications@github.com>> wrote:


Sorry I made a typo:

PiFaceDigital


with a lower-case i after first P.

More examples are in the docs: http://pifacedigitalio.readthedocs.org/example.html
